src/strategy.coffee

```diff
@@ -13,19 +13,25 @@ module.exports = (oAuth2Strategy) ->
       options.authorizationURL ?= constants.defaultAuthorizationURL
       options.tokenURL ?= constants.defaultTokenURL
       options.verifyURL ?= constants.defaultVerifyURL
+      @params = new Object()
+      @params.scope = options.scope or ''
+
       @_verifyURL = options.verifyURL
 
       super(options, @_verifyOAuth2)
       @_oauth2.useAuthorizationHeaderforGET(true)
 
       @name = 'eveonline'
 
+    authorizationParams: () ->

```

Comment:
Unfortunately we can't change the name of this function as passport-eveonline extends the passport-oauth2 class, authorizationParams gets called by the internals of passport-oauth2:
https://github.com/jaredhanson/passport-oauth2/blob/master/lib/strategy.js#L216

By default this is an empty object:
https://github.com/jaredhanson/passport-oauth2/blob/master/lib/strategy.js#L291

passport-oauth2 then extends the returned object to construct the uri for the authorization request:
https://github.com/jaredhanson/passport-oauth2/blob/master/lib/strategy.js#L233-L234
https://github.com/jaredhanson/passport-oauth2/blob/master/lib/strategy.js#L244-L245